src/index.ts

```diff
@@ -1,20 +1,21 @@
-export * from './allPass';
-export * from './fromPairs';
-export * from './toPairs';
-export * from './xprod';
+export * from './add';
 export * from './adjust';
+export * from './all';
+export * from './allPass';
 export * from './always';
 export * from './and';
+export * from './any';
+export * from './anyPass';
 export * from './append';
 export * from './assoc';
-export * from './takeLast';
-export * from './endsWith';
 export * from './clamp';
 export * from './defaultTo';
-export * from './prop';
-export * from './anyPass';
-export * from './any';
-export * from './all';
+export * from './endsWith';
 export * from './equals';
+export * from './fromPairs';
+export * from './ifElse';
 export * from './objOf';
-export * from './add';
+export * from './prop';
+export * from './takeLast';
+export * from './toPairs';
+export * from './xprod';
```

Comment:
many "unrelated" changes here, do you have some sort-imports plugin enabled? Can I ask to remove these changes?